CHANGELOG/CHANGELOG-1.21.md

```diff
@@ -197,12 +197,7 @@ filename | sha512 hash
 - Cloud-controller-manager: routes controller should not depend on --allocate-node-cidrs ([#97029](https://github.com/kubernetes/kubernetes/pull/97029), [@andrewsykim](https://github.com/andrewsykim)) [SIG Cloud Provider and Testing]
 - Copy annotations with empty value when deployment rolls back ([#94858](https://github.com/kubernetes/kubernetes/pull/94858), [@waynepeking348](https://github.com/waynepeking348)) [SIG Apps]
 - Detach volumes from vSphere nodes not tracked by attach-detach controller ([#96689](https://github.com/kubernetes/kubernetes/pull/96689), [@gnufied](https://github.com/gnufied)) [SIG Cloud Provider and Storage]
-- Error: You must provide one or more resources by argument or filename.
-  Example resource specifications include:
-     '-f rsrc.yaml'
-     '--filename=rsrc.json'
-     '<resource> <name>'
-     '<resource>' ([#97440](https://github.com/kubernetes/kubernetes/pull/97440), [@pandaamanda](https://github.com/pandaamanda)) [SIG CLI]
+- Fix kubectl label error when local=true is set. ([#97440](https://github.com/kubernetes/kubernetes/pull/97440), [@pandaamanda](https://github.com/pandaamanda)) [SIG CLI]
```

Comment:
While it seems fine to me changing it here, we also have to take the release notes process into account for unpublished minor releases. 

I see that the PR https://github.com/kubernetes/kubernetes/pull/97440 is already updated. So the next generation of the release notes will pick those changes up.

cc @wilsonehusin @pandaamanda

Reply:
commented on why this is happening: https://github.com/kubernetes/kubernetes/pull/97440#issuecomment-760085627

Reply:
Got it. :+1:

Reply:
Fixed.